src/views/studio/studio-tab-nav.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {useRouteMatch, NavLink} from 'react-router-dom';
+
+const StudioTabNav = () => {
+    const match = useRouteMatch();
+
+    return (
+        <div>
+            <NavLink
+                activeStyle={{textDecoration: 'underline'}}
+                to={`${match.url}`}
+                exact
+            >
+                Projects
+            </NavLink>
+            &nbsp;|&nbsp;
+            <NavLink
+                activeStyle={{textDecoration: 'underline'}}
+                to={`${match.url}/curators`}
+            >
+                Curators
```

Comment:
Just noticed that the order of the tabs is different from the current studios page (not that it really matters for this skeleton)